integration

```diff
@@ -1,6 +1,7 @@
 [
   "0xAlon/dolphin",
   "0xAlon/tami4edge",
+  "3ll3d00d/jriver_homeassistant",
```

Comment:
actions failing.

Reply:
published updated release fixing those identified gaps & updated links in original post in PR